languages/python/omni_python/src/omni_python/dbapi.py

```diff
@@ -0,0 +1,361 @@
+"""
```

Comment:
I think it would be prudent to include something like: "Based on ...." (as we're expected to evolve this) and include license terms

https://github.com/petere/plpydbapi/blob/master/LICENSE

Reply:
Added the original license